master/getting-started/rkt/tutorials/basic.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Calico networking with rkt
+---
+
+
+This tutorial describes how to set up a Calico cluster in a pure rkt environment
+using CNI with Calico specific network and IPAM drivers.
+
+Using the Calico CNI plugin, the required network setup and configuration
```

Comment:
I think we can probably get rid of these two paragraphs.

No need to go into the details of our CNI plugins and how they work.  This is meant to be a quick start, so we should cut out as much text as we can!